### PR TITLE
[FRONT] - Refactoring: change palette color as figma

### LINKED ISSUE #120: [FRONT]  - Refactoring:  Change color palete
It is necessary to change the color of the palette as in figma

### PR DESCRIPTION
**Pull request by m83city**
change pallete color

### CODE REVIEW

File: app/src/main/res/values/styles.xml [hunk 1]
```diff
@@ -1,6 +1,22 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
 
+
+    <style name="StyleTextInputLayout" parent="TextAppearance.AppCompat">
```
Comment (vlad-vinskevitch) [inline app/src/main/res/values/styles.xml]: rename StyleTextInputLayout -> text_field_style

File: app/src/main/res/values-night/color.xml [hunk 2]
```diff
@@ -1,14 +1,15 @@
...
+    <color name="Primary_100">#DFE4F1</color>
+    <color name="Primary_300">#CCD0DC</color>
+    <color name="Primary_500">#B8BCC7</color>
+    <color name="Primary_700">#7A7D81</color>
+
     <color name="Secondary">#311b92</color>
-    <color name="Background"></color>
     <color name="Error">#d32f2f</color>
 
     <color name="OnError">#fff</color>
```
Comment (vlad-vinskevitch) [inline app/src/main/res/values-night/color.xml]: dont use camelCase, use only small litteral and underscope "on_error"

File: app/src/main/res/values-night/color.xml [hunk 3]
```diff
@@ -1,14 +1,15 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <color name="Transparent">#00000000</color>
 
-    <color name="Primary">#536dfe</color>
+    <color name="Primary_100">#DFE4F1</color>
```
Comment (vlad-vinskevitch) [inline app/src/main/res/values-night/color.xml]: use only small litteral and underscope in .xml

**Review by vlad-vinskevitch — CHANGES_REQUESTED** [review]

File: app/src/main/res/values/styles.xml (hunk 1 above)
Comment (m83city) [reply app/src/main/res/values/styles.xml]: I'm refactor code because it non actual

File: app/src/main/res/values-night/color.xml (hunk 2 above)
Comment (m83city) [reply app/src/main/res/values-night/color.xml]: ok

File: app/src/main/res/values-night/color.xml (hunk 3 above)
Comment (m83city) [reply app/src/main/res/values-night/color.xml]: ok

**Review by vlad-vinskevitch — APPROVED** [review]

**Pull request merged**